wsd/PresetsInstall.hpp

```diff
@@ -26,7 +26,7 @@ class PresetsInstallTask : public std::enable_shared_from_this<PresetsInstallTas
     std::string _configId;
     std::string _presetsPath;
     std::vector<std::function<void(bool)>> _installFinishedCBs;
-    SocketPoll& _poll;
+    std::weak_ptr<SocketPoll> _poll;
```

Comment:
This looks promising - but ... I'm fairly sure that I massively boobed by not reviewing the design of this preset thing early. I expect we need to re-visit this interactively at some length.